airflow-core/src/airflow/ui/public/i18n/locales/ar/_freeze_exemptions.json

```diff
@@ -0,0 +1,22 @@
+{
+  "admin": {},
+  "assets": {},
+  "browse": {},
+  "common": {},
+  "components": {},
+  "dag": {},
+  "dags": {},
+  "dashboard": {},
+  "hitl": {
```

Comment:
This section will be reunited with `hitl.json` (I'm aware of the duplications - they will later be removed)